Sources/App/Views/PackageController/PackageShowView.swift

```diff
@@ -198,14 +95,21 @@ extension PackageShowView {
         let history: History?
         let activity: Activity?
         let products: ProductCounts?
+        let releases: ReleaseInfo
+        let languagePlatforms: LanguagePlatformInfo
 
         struct Link: Equatable {
             let name: String
             let url: String
         }
 
+        struct DatedLink: Equatable {
+            let date: String   // FIXME: use RelativeDateTimeFormatter
+            let link: Link
+        }
+
         struct History: Equatable {
-            let since: String  // TODO: use Date and derive on the fly
+            let since: String  // FIXME: use RelativeDateTimeFormatter
```

Comment:
I'll convert these to `Date` and tackle the relative dates when I actually populate them from data.